scripts/scripts.js

```diff
@@ -200,6 +200,37 @@ export function formatDate(date) {
   });
 }
 
+function addBlogTopics(main) {
+  const sidebarBolgTopic = main.querySelector('.blog-topics');
+  const apiUrl = '/blogs/query-index.json?sheet=topics';
+  if (sidebarBolgTopic) {
+    fetch(apiUrl)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+        return response.json();
+      })
+      .then((data) => {
+        const blogTitles = data.data;
+        blogTitles.forEach((item) => {
+          const pTag = document.createElement('p');
+          pTag.setAttribute('class', 'button-container');
+          const aTag = document.createElement('a');
+          aTag.setAttribute('href', item.path);
+          aTag.setAttribute('title', item.header);
+          const textNode = document.createTextNode(item.header);
+          aTag.appendChild(textNode);
+          pTag.appendChild(aTag);
+          sidebarBolgTopic.appendChild(pTag);
```

Comment:
Just as a suggestion, I added to the project a very small JS here: https://github.com/hlxsites/servicenow/blob/main/scripts/dom-helpers.js which allows you to write this in a more compact and easier to read form.

```javascript
sidebarBlogTopic.append(
  ...blogTopics.map((topic) => (
    p({class: 'button-container'},
      a({ href: topic.path, title: topic.header }, topic.header)
    )
  )),
);
```

in my opinion, this is easier to read later and see exactly what will be the structure of the DOM generated, but as said is just a suggestion, as writing it as you did here is perfectly ok.